example/src/components/Options.tsx

```diff
@@ -1,81 +1,153 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { View, Text, StyleSheet } from 'react-native';
-import { AllowedTypes, VerificationSessionOptions } from '../types';
+import {
+  AllowedTypes,
+  VerificationSessionOptions,
+  VerificationType,
+} from '../types';
 import { Option } from './Option';
+import RadioGroup from 'react-native-radio-buttons-group';
+import type { RadioButtonProps } from 'react-native-radio-buttons-group';
 
 type OptionsProps = {
   options: VerificationSessionOptions;
   setOptions(options: VerificationSessionOptions): void;
 };
 
 export function Options({ options, setOptions }: OptionsProps) {
-  return (
-    <View style={styles.container}>
-      <View>
-        <Text style={styles.label}>Allowed Types:</Text>
-        <View style={styles.section}>
-          <Option
-            title="Driving License"
-            value={options.allowedTypes[AllowedTypes.DRIVING_LICENSE]}
-            onChange={(value) =>
-              setOptions({
-                ...options,
-                allowedTypes: {
-                  ...options.allowedTypes,
-                  [AllowedTypes.DRIVING_LICENSE]: value,
-                },
-              })
-            }
-          />
-          <Option
-            title="Passport"
-            value={options.allowedTypes[AllowedTypes.PASSPORT]}
-            onChange={(value) =>
-              setOptions({
-                ...options,
-                allowedTypes: {
-                  ...options.allowedTypes,
-                  [AllowedTypes.PASSPORT]: value,
-                },
-              })
-            }
-          />
-          <Option
-            title="ID Card"
-            value={options.allowedTypes[AllowedTypes.ID_CARD]}
-            onChange={(value) =>
-              setOptions({
-                ...options,
-                allowedTypes: {
-                  ...options.allowedTypes,
-                  [AllowedTypes.ID_CARD]: value,
-                },
-              })
-            }
+  const [verificationTypeRadioButtons] = useState([
```

Comment:
This array should be a constant variable instead of a state.

Reply:
hmm I tried to replace it with a const, but the UI is not behaving correctly, see below(The [refererence](https://www.npmjs.com/package/react-native-radio-buttons-group) to official API is also using a state)

| Before  | After |
| ------------- | ------------- |
| ![beforeChagne](https://user-images.githubusercontent.com/79880926/227663810-3dac4aea-243c-48b9-ae7a-14fa03671dce.gif) | ![afterChange](https://user-images.githubusercontent.com/79880926/227663808-68dfe621-a288-4ad3-be42-b7a83095d655.gif) |